cmd/lakectl/cmd/root.go

```diff
@@ -94,6 +94,12 @@ type Configuration struct {
 		// setting FixSparkPlaceholder to true will change spark placeholder with the actual location. for more information see https://github.com/treeverse/lakeFS/issues/2213
 		FixSparkPlaceholder bool `mapstructure:"fix_spark_placeholder"`
 	}
+	// Experimental - Use caution when enabling experimental features. It should only be used after consulting with the lakeFS team!
```

Comment:
Not sure about the "consult" part - 
Might invoke Slack messages about it, and exposing it more.

Reply:
We have to do our due diligence and warn against use of experimental features without consulting with us first.
It is better than the alternative